dataland-frontend/src/components/pages/SearchCompaniesForFrameworkData.vue

```diff
@@ -12,35 +12,50 @@
           <FrameworkDataSearchBar
```

Comment:
Focus on this does not work anymore when switching between toggled view and scrolltop-view